src/store/drawing/reducer.ts

```diff
@@ -7,14 +8,21 @@ import {
     redoDrawingPathGroupAction,
     resetDrawingAction,
     setDrawingLocationAction,
+    setDrawingStrokeAction,
     undoDrawingPathGroupAction,
 } from './actions';
 import { DrawingState } from './types';
 
+export const defaultStroke = {
```

Comment:
Does this need to be exported separately? It seems like we'd only want to access the current state and have this be encapsulated in the reducer.